src/commands/kv/namespace/site.rs

```diff
@@ -41,7 +41,12 @@ pub fn site(
 
                     get_id_from_namespace_list(&client, target, &title)
                 } else {
-                    failure::bail!("{:?}", api_errors.errors)
+                    let error_messages: Vec<String> = api_errors
+                        .errors
+                        .iter()
+                        .map(|e| e.message.clone())
+                        .collect();
+                    failure::bail!("ApiError {:?}", error_messages);
```

Comment:
oh actually - it looks like we already have a convenience function for this that should make it easier + uniform with the way we display errors.

It is `crate::http::format_error`, and it takes an `ApiFailure`, and an optional function to handle individual error codes.

So you could write something real quick to include the suggestions for codes `10026` and `10014` and pass it to that function. You can see an example of this in `crate::src::commands::route::delete`, at the end there is a `match result` and it calls `http::format_error`

This should allow us to display the error without making it look like an array

Reply:
(if i had thought of this before i would have pointed you in that direction!)